rmc/pipeline/two_breaks/run_batches.py

```diff
@@ -609,23 +601,13 @@ def main(args):
 
     if not args.docker_image:
         logger.info("Picking default docker image...")
-        # Use a docker image that specifies spark memory allocation if --use-custom-machine was specified
-        if args.use_custom_machine:
-            args.docker_image = "gcr.io/broad-mpg-gnomad/rmc:20220930"
-            # Hail versions >= 0.2.119 are no longer backwards compatible
-            # (older Hail versions cannot read data written using version >= 0.2.119)
-            raise DataException(f"Hail in {args.docker_image} image is out of date!")
-        # Otherwise, use the default docker image
-        else:
-            args.docker_image = (
-                "us-central1-docker.pkg.dev/broad-mpg-gnomad/images/rmc_simul_search"
-            )
-            # NOTE: Python version on your local machine must match the Python version in this image
-            logger.warning(
-                "Using %s image; please make sure Hail and Python versions in image are"
-                " up to date",
-                args.docker_image,
-            )
+        args.docker_image = "hailgenetics/hail:0.2.132-py3.11"
```

Comment:
The recommended image has `py3.10` in line 844

Reply:
oops!